modules/nw-ovn-ipsec-certificates.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-ovn-ipsec-certificates_{context}"]
+= Security certificate generation and rotation
+
+The Cluster Network Operator (CNO) generates a self-signed X.509 certificate authority (CA) that is used by IPsec for encryption. Certificate signing requests (CSRs) from each node are automatically fulfilled by the CNO.
+
+The CA is valid for 10 years. The individual node certificates are valid for 5 years and are automatically rotated after 4 and a half years elapse.
```

Comment:
I wouldn't spell out 4-and-a-half years. 
"after 4 1/2 years elapse" is a more standardized convention.

Reply:
@bobfuru, so you're saying I'm nonstandard now? :)